### DISCUSSION TITLE
二维码无法被扫描

### DISCUSSION THREAD

**Discussion by Skyline-R** [Richasy/Bili.Uwp#1653] [问答]
出问题的设备是三星galaxybook2，windows11，大约12寸的屏幕
- 在别的设备上（台式机上）使用这款app，登录二维码可以正常被扫描
- 把二维码截图出来发到台式机显示器上也无法扫描
- 不是手机摄像头问题，用ipad扫也没有办法，拍成照片扫会提示未解析出二维码

**Comment by ChoHope** [comment]
我的设备MateBook E Go， Windows 11 家庭中文版，进入软件后用账号密码登陆时提示“密码登录方式失败，请使用二维码登录。”然后用手机端扫描二维码，也是一直扫描不出来结果。
